apps/backend/src/utils/ssh_command_manager.py

```
@@ -0,0 +1,481 @@
+        # All retries failed
+        raise SSHCommandError(
+            message=f"Command {command_name} failed after {command_def.retry_count} attempts: {last_exception}",
+            command=formatted_command,
+            hostname=connection_info.host
+        ) from last_exception
+    
+    async def execute_raw_command(
+        self,
+        command: str,
+        connection_info: SSHConnectionInfo,
+        timeout: int = 30,
```

Ruff (0.12.2)

Unused method argument: `timeout`

(ARG002)